### ISSUE TITLE
Is there any chance that a clip can't be recovered

### ISSUE THREAD

**Issue by wl03355609** [TwitchRecover/TwitchRecover#20]
So I saw someone clipped something
The stream url is https://streamscharts.com/twitch/channels/steveogb/streams/40750129996
The tool can't recover it
And from other streamer seems that some clips from me and got deleted aren't recoverable
So is this a bug or it works like this?

**Comment by daylamtayari** [comment]
So how it currently works is that it checks all of the possible clip URL combinations for a particular stream and returns the URLs for which there was a clip/MP4 file attached to.
That part of the program works fine without issues afaik at this point but I am going to look into this in case Twitch changed something which may have broken it for some streamers/some streams.
I have just tested it and can still fully recover clips all of the clips from a stream from yesterday which had its VOD and all of its clips deleted so I don't think that it is Twitch that broke something significant but I am still going to look into this.

**Comment by wl03355609** [comment]
Yeah, I know there's some stream can be recovered even the streamer don't kept the vod
So it's odd somehow

**Comment by daylamtayari** [comment]
>  So it's odd somehow  

LULW welcome to trying to reverse engineer Twitch hahaha
